### PR TITLE
Solution for issue #264

### PR DESCRIPTION
**Pull request by renato04**
Issue #264

Compare Maybe<T> with null value using `==` operator returns `true`

### CODE REVIEW

**Review by petrechitashvili — APPROVED** [review]
Looks good to me! 🙂

**Comment by renato04** [comment]
Ops, sorry for the miss click

File: CSharpFunctionalExtensions/Maybe/Maybe.cs [hunk 1]
```diff
@@ -48,7 +48,10 @@ public static Maybe<T> From(T obj)
                 return maybe.Equals(value);
 
             if (maybe.HasNoValue)
-                return false;
+                if (ReferenceEquals(value, null))
```
Comment (hankovich) [inline CSharpFunctionalExtensions/Maybe/Maybe.cs:51]: can be just 
```csharp
if (maybe.HasNoValue)
    return value is null;
```

**Comment by vkhorikov** [comment]
Merging this, will be published soon.

**Pull request merged**